src/app/app.component.ts

```diff
@@ -15,6 +15,6 @@ export class AppComponent {
 
   public logout(): void {
     this.authenticationService.logout();
```

Comment:
Que diferencia tiene `location.reload(true);` vs `this.router.navigate(['/login'])` ?

Reply:
location.reload(true) hace que la página se recargue, entonces impide el flow de la aplicación, cuando cambias por router cambia a un componente ya creado, entonces no recarga la página, es sobre todo para mejorar ux.

Reply:
Me parece bien 👍